test/LPStaking.test.js

```diff
@@ -22,23 +22,31 @@ contract("LPStaking", accounts => {
         token = await Token.deployed();
         lp_token = await LPToken.new();
 
-        factory = await LPFactory.deployed();
+        factory = await StakingFactory.deployed();
     })
 
     it("check token", async () => {
         let balance = await token.balanceOf(owner);
 
         expect(balance.toString()).to.equal(
-            web3.utils.toWei('20000000'));
+            web3.utils.toWei('1000000000'));
     })
     
     it('new lp staking pool - 1', async () => {
-        await factory.newStakingPool(lp_token.address, 100);
-        pool1 = await LPStaking.at(await factory.stakingPools(0));
+        await factory.newLPStakingPool(lp_token.address);
+        pool1 = await LPStaking.at(await factory.lpStakingPools(0));
         expect(pool1.address).to.not.equal(0x0000000000000000000000000000000000000000);
+    })
+
+    it('distribute 10000 tokens - reward rate should be 10000*10**18/(rewardDuration)', async () => {
+        await token.approve(pool1.address, web3.utils.toWei('10000'), {from: owner});
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/tranche-jibrel/new-staking/pullRequest?prid=8459503)
```suggestion
        await token.approve(pool1.address, web3.utils.toWei("10000"), {from: owner});
```